ngraph_onnx/onnx_importer/utils/pool.py

```diff
@@ -26,30 +26,31 @@
 import logging
 
 from ngraph_onnx.onnx_importer.utils.conv import get_pads, get_strides, get_kernel_shape
-from ngraph_onnx.onnx_importer.utils.decorators import refactoring_required
 
 if TYPE_CHECKING:
     from ngraph_onnx.onnx_importer.model_wrappers import NodeWrapper
 
 log = logging.getLogger(__file__)
 
 
-def get_op_type(onnx_node):  # type: (NodeWrapper) -> str
+def get_op_type(onnx_node):  # type: (NodeWrapper) -> Tuple[str, bool]
     """
     Parse ONNX pooling operation attributes and produce an ngraph compatible pool_params dict.
 
     :param onnx_node: wrapped ONNX node for a pooling operation op
-    :return: dict of pool_params for ng.pooling
+    :return: dict of pool_params for ng.pooling and if it is a global pooling op
```

Comment:
Update return value to conform with what is stated in the type annotation: Tuple[str, bool]. There is no dict mentioned.